misc/userscripts/cast

```diff
@@ -133,17 +133,23 @@ echo "jseval -q $(printjs)" >> "$QUTE_FIFO"
 
 tmpdir=$(mktemp -d)
 file_to_cast=${tmpdir}/qutecast
+program_=$(which castnow)
```

Comment:
which isn't portable, I think it would be better to use the bash builtin `program_="$(command -v castnow)"`

Reply:
Nice. I didn't know about that. Thanks.